routes/recordOverview.js

```diff
@@ -31,6 +34,138 @@ var route = function (req, res, next) {
 
         },
 
+        //set detail content to overviewDetail
+        function (cb) {
+
+            //make sure form is object and is not empty
+            if (typeof req.linz.model.linz.formtools.overview.details.form !== 'object' || utils.isEmptyObject(req.linz.model.linz.formtools.overview.details.form)) {
+                return cb(null);
+            }
+
+            linz.formtools.form.generateFormFromModel(req.linz.model.schema, req.linz.model.linz.formtools.overview.details.form, req.linz.record, 'edit', function (err, viewForm) {
+
+                if (err) {
+                    return cb(err);
+                }
+
+                var detailForm = viewForm.render();
+
+                res.render(linz.api.views.viewPath('recordOverviewDetails.jade'), {
+                    form: detailForm
+                }, function (renderErr, html) {
+
+                    if (renderErr) {
+                        return cb(renderErr);
+                    }
+
+                    locals.overviewDetail = html;
+                    return cb(null);
+                });
+
+            });
+        },
+
+        //process tabs of overview detials
+        function (cb) {
+
+            if (!Array.isArray(req.linz.model.linz.formtools.overview.details.tabsArray) || !req.linz.model.linz.formtools.overview.details.tabsArray.length) {
+                return cb(null);
+            }
+
+            (function processTabs(i) {
+
+                var detailTab = req.linz.model.linz.formtools.overview.details.tabsArray[i];
+                var tempTab = {};
+                tempTab.label = detailTab.label;
+
+                if (typeof detailTab.form === 'object') {
```

Comment:
I'm not a big fan of the `if`, `else if`, `else` pattern. Could this please be rewritten. It's not used much in Linz.js anywhere. If you peruse the source code, you should find some better alternatives (such as `return` early).